src/VscPreviewExtension/media/buildTree.js

```diff
@@ -0,0 +1,319 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+function buildTree(containerName, treeData) {
+  var i = 0;
+  var durationTime = 750;
```

Comment:
no, better naming

---
In reply to: [80399940](https://github.com/dotnet/docfx/pull/693#discussion_r80399940) [](ancestors = 80399940,80396814)